README.md

```diff
@@ -7,10 +7,9 @@ Kindly is an open source web application created by Trafalgar Girls and enBloc,
 
 <h2 align=center>Contributing</h2>
 
-We **welcome** your contributions. Please familiarise yourself with the following documentation before getting started. Contributing docs can be found in [kindly/.github](https://github.com/enBloc-org/kindly/tree/dev/.github)
-
-You can also connect with the developing community on our [discord server](https://discord.gg/Mrh5NYw52S)
+We **welcome** your contributions. Please familiarize yourself with our documentation before getting started. You can find guidance on how to contributing in the following sources:
```

Comment:
You can find guidance on how to contributing* in the following sources:

*contribute

Reply:
Fixed in a new commit 👍